### ISSUE TITLE
streamline WD item creation process

### ISSUE THREAD

**Issue by VladimirAlexiev** [diegodlh/zotero-cita#195]
I understand the desire not to create duplicate items in WD, but the  WD item creation process is difficult:
- I can only add WD items one by one
- it asks me 3 (!) confirmation questions before it proceeds with WD creation
- coupled with #194, it makes the process ridiculously inefficient

Please:
- (Optionally, enable creation as a separate operation, not necessarily after "Fetch QID(s)"
- Enable creating multiple items at once: eg for all items where no QID could be fetched
- Only ask 1 confirmation
- Add a checkbox "[ ] Do not ask for confirmation again"

**Comment by diegodlh** [comment]
Hi, @VladimirAlexiev. Thanks for your comments.

In most cases, creating Wikidata items from Cita is not as straightforward as it could be on purpose, because that's not the main goal of this plugin. Other workflows, such as using the Wikidata QuickStatements export translator, are recommended for these batch creation use cases: https://www.wikidata.org/wiki/Wikidata:Zotero.

Nonetheless, I agree that two of your proposals may be worth considering:
> * Enable creating multiple items at once: eg for all items where no QID could be fetched
> * Add a checkbox "[ ] Do not ask for confirmation again"

**Comment by Futur3r** [comment]
On the same subject,
would it be possible to have an "Export citations to quickstatements" like the "Export to file" option in "More" in the Citations tab ?
That functionality would put the citations without QID (as the "Export to file" option with QS translator already do) in the clipboard to be used in Calc or directly in quickstatements ?

It can be already be done but as stated in the first comment of this issue, it add to much extra steps.

**Comment by Dominic-DallOsto** [comment]
Yeah, that should be fairly easy to implement.

So what you get from exporting to a file with the quick statements format is what you want? Just if that could copy to clipboard in one step it would be nicer?

**Comment by Futur3r** [comment]
Yes, because every time I export an item or a citation with the QS translator it create a file. And then I need to copy what's in that file to paste it in a speadsheet.

If at least the translator would export directly in the clipboard, it would be waay better !

**Comment by Futur3r** [comment]
Another idea, as it was said before, sure Cita isn't meant first for contributing bibliographic data into Wikidata. But for a Wikidata contributor it's a gold mine. Most of the tools to contribute in a 'semi-automated' way are there and it's easier and less time consuming to control bibliographic data quality with Zotero before uploading into Wikidata.

So as a compromise, I propose a setting in the preferences window, where we could enable an "Advanced mode" to allow the functionalities stated before and other functionalities to make contribution into Wikidata almost only with Cita.
Another good thing I see, is that when you contribute with Cita, it is stated as so in the journal of an element. So it would be easier to detect a potential problem with Cita, to correct it, and also it would be clearer to patrol and potentially revert.
